website/content/configuration/_index.md

```diff
@@ -4,28 +4,53 @@ weight: 4
 ---
 
 MetalLB remains idle until configured. This is accomplished by
-creating and deploying a config map into the same namespace
+creating and deploying various resources into the same namespace
 (metallb-system) as the deployment.
 
-There is an example config map in
-[`config/samples/example-config.yaml`](https://raw.githubusercontent.com/metallb/metallb/main/config/samples/example-config.yaml),
+There are various examples of the configuration CRs in
+[`config/samples`](https://raw.githubusercontent.com/metallb/metallb/main/config/samples),
 annotated with explanatory comments.
 
-If you've named the config map `config.yaml`, you can deploy the manifest with `kubectl apply -f config.yaml`.
+Also, the API is [fully documented here](../apis/_index.md).
 
-{{% notice note %}}
```

Comment:
same reason^^ I'd mention here again that the resources need to be in the same namespace as the deployment